config/locales/devise/en.yml

```diff
@@ -104,46 +104,45 @@ en:
       already_signed_out: You are now signed out.
     two_factor_authentication:
       contact_administrator: Please contact your system administrator.
-      header_text: Enter the passcode we sent you
+      header_text: Enter your passcode
       invalid_otp: >
-        That one-time passcode is invalid. Please try again or request a new
+        That passcode is invalid. You can try entering it again or request a new
         one-time passcode.
-      invalid_recovery_code: Invalid recovery code. Please try again.
+      invalid_recovery_code: That recovery code is invalid.
```

Comment:
I can't think of anything else. Here is Apple's support article on the subject for any ideas: https://support.apple.com/en-us/HT202649